db/migrate/20160801053501_add_confirmed_at_to_users.rb

```diff
@@ -0,0 +1,8 @@
+class AddConfirmedAtToUsers < ActiveRecord::Migration[5.0]
```

Comment:
What happens if an account existed before these columns were created? Will they be able to log in?